docs/source/run_locally/llama.cpp.rst

```diff
@@ -55,14 +55,14 @@ Then you can run the model with the following command:
 
 .. code:: bash
 
-   ./main -m qwen2-7b-instruct-q5_k_m.gguf -n 512 --color -i -cml -f prompts/chat-with-qwen.txt
+   ./llama-cli -m qwen2-7b-instruct-q5_k_m.gguf -n 512 --color -i -cml -f prompts/chat-with-qwen.txt
```

Comment:
I believe they have also deprecated -cml. Could you also transfer the docs at https://huggingface.co/Qwen/Qwen2-72B-Instruct-GGUF here

Reply:
sure fixed it